pyproject.toml

```diff
@@ -50,6 +50,7 @@ vendoring = {version = "^0.3", python = "^3.8"}
 pep517 = "^0.11.0"
 black = {version = "^21.9b0", markers = "python_version >= '3.6.2' and python_version < '4.0' and implementation_name != 'pypy'"}
 isort = {version = "^5.9.3", python = "^3.6.1"}
+mypy = ">=0.931"
```

Comment:
While I agree WIBNI these were in one place only, I am not sure that I am convinced that removing them from `pyproject.toml` is a good idea, though.

Just as a matter of workflow: after running `poetry install` in this project I'd sure like to be in an environment that has all the dev tools that I'll be using.

But also precommit is suboptimal, certainly in mypy, because it does not run in the correct virtual environment (per https://github.com/python-poetry/poetry/pull/4510#issuecomment-968125631).  

I wonder if the same is true for isort, which also behaves differently according to the environment.